src/opm/input/eclipse/share/keywords/000_Eclipse100/R/RADFIN

```diff
@@ -6,49 +6,73 @@
   "size": 1,
   "items": [
     {
+      "item": 1,
       "name": "NAME",
       "value_type": "STRING"
     },
     {
-      "name": "I1",
+      "item": 2,
+      "name": "I",
       "value_type": "INT"
     },
     {
-      "name": "I2",
-      "value_type": "INT"
-    },
-    {
-      "name": "J1",
-      "value_type": "INT"
-    },
-    {
-      "name": "J2",
+      "item": 3,
+      "name": "J",
       "value_type": "INT"
     },
     {
+      "item": 4,
       "name": "K1",
       "value_type": "INT"
     },
     {
+      "item": 5,
       "name": "K2",
       "value_type": "INT"
     },
     {
+      "item": 6,
       "name": "NR",
       "value_type": "INT"
     },
     {
+      "item": 7,
       "name": "NTHETA",
       "value_type": "INT"
     },
     {
+      "item": 8,
       "name": "NZ",
       "value_type": "INT"
     },
     {
+      "item": 9,
       "name": "NWMAX",
       "value_type": "INT",
       "default": 1
+    },
+    {
+      "item": 10,
+      "name": "INNER_RADIUS",
+      "value_type": "DOUBLE",
+      "default": 0.1524
```

Comment:
When you are here - items 10-12 should all have `"dimension" : "Length"`

Reply:
> Items 10-12 should all have `"dimension" : "Length"`

Good point.  I've pushed an update to that effect.